applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -390,10 +398,13 @@ void MainWindow::createPluginControls(QSharedPointer<ANSHAREDLIB::PluginManager>
     setTabPosition(Qt::RightDockWidgetArea,QTabWidget::East);
     setDockOptions(QMainWindow::ForceTabbedDocks);
 
+    pPluginManager->setParent(this);
     //Add Plugin controls to the MainWindow
     for(IPlugin* pPlugin : pPluginManager->getPlugins()) {
         if(QDockWidget* pControl = pPlugin->getControl()) {
             addDockWidget(Qt::LeftDockWidgetArea, pControl);
+            pControl->setParent(this);
```

Comment:
Do we need this if we use the `setAttribute(Qt::WA_DeleteOnClose)` solution from above?